api/src/pcapi/core/educational/validation.py

```diff
@@ -125,3 +129,19 @@ def check_user_can_prebook_collective_stock(uai: str, stock: models.CollectiveSt
 def check_institution_id_exists(institution_id: int) -> None:
     if not models.EducationalInstitution.query.get(institution_id):
         raise exceptions.EducationalInstitutionNotFound()
+
+
+def check_if_offer_not_used_or_reimbursed(
+    offer: Offer | CollectiveOffer | CollectiveOfferTemplate,
+) -> None:
+    if isinstance(offer, (Offer, CollectiveOffer)):
+        stocks = offer.collectiveStock
```

Comment:
(détail / nommage) L'utilisation du pluriel semble indiquer qu'il y a plusieurs éléments, alors que ce n'est pas le cas.

```suggestion
        stock = offer.collectiveStock
```